src/styles.css

```diff
@@ -0,0 +1,1204 @@
+}
+
+.overlay a {
+  padding: 8px;
+  text-decoration: none;
+  font-size: 32px;
+  color: #fff;
+  font-weight: 600;
+  display: block;
+  transition: 0.3s;
+  line-height: 44px;
+}
+
+.overlay a:hover,
+.overlay a:focus {
+  color: #f1f1f1;
+}
+
+.overlay .closebtn {
+  position: absolute;
+  top: 11px;
+  right: 5px;
+  font-size: 30px;
+  font-weight: normal;
+  border: 2px;
+  cursor: pointer;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  background-image: url("../assets/header.png");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-color: #fff;
+  width: 100%;
+  height: 90vh;
+  border-radius: 0 0 0 100px;
+  border-bottom: 2px solid #cfd8dc;
+}
+
+.intro {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  width: 70%;
+  margin: 5%;
+  margin-top: 25%;
+  animation-name: introin;
+  animation-duration: 4s;
+}
+
+@keyframes introin {
+  from {
+    margin-left: 100%;
+  }
+
+  to {
+    margin-left: 22%;
+  }
+}
+
+textarea::placeholder {
+  color: #b3bac5;
+  padding-top: 15px;
+  font-size: 15px;
+}
+
+input[type="name"]::placeholder {
+  color: #172b4d;
+  font-size: 17px;
+}
+
+input[type="email"]::placeholder {
+  color: #172b4d;
+  font-size: 17px;
+}
+
+.intro1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  position: static;
+  width: 90%;
+  height: 122px;
+  left: 24px;
+  top: 276px;
+  color: #344563;
+  margin-top: 0;
+  margin-top: -20px;
+}
+
+p.intro1 {
+  padding: 15px;
+}
+
+.intro2 {
+  color: #7f8cff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+  display: flex;
+  align-items: flex-start;
+  padding-left: 15px;
+  grid-gap: 30px;
+  justify-items: center;
+  margin-left: 20%;
+}
+
+.intro3 {
+  color: #7f8cff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+  display: flex;
+  align-items: flex-start;
+  padding-left: 15px;
+  grid-gap: 30px;
+  justify-items: center;
+  margin-left: 2%;
+}
+
+.social {
+  display: flex;
+  list-style: none;
+}
+
+ul.social {
+  margin-block-start: 1px;
+  margin-block-end: 1px;
+  padding-inline-start: 0;
+}
+
+.social-icon {
+  align-items: center;
+  justify-content: center;
+  height: 20px;
+  padding: 10px 0 0 20px;
+  transition: all 2s;
+}
+
+.social-icon:hover {
+  transform: rotate(360deg);
+}
+
+li.social-icon {
+  display: flex;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  padding: 14px;
+  width: 97%;
+  height: 600px;
+  left: calc(50% - 327px / 2);
+  top: 114px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 2%;
+}
```

Comment:
Please ensure your pop-up design matches that of the given Figma design.

## Mobile popup
Your design | Figma design
![Screenshot from 2023-07-26 22-54-24](https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/44624138/20ad458b-a23a-489f-9cf0-8c234c5f41d3)  |  ![Screenshot from 2023-07-26 22-59-52](https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/44624138/f708edd2-0eb5-4576-b4c1-2a31ef7b72d3)

## Desktop design
Your design | Figma design
![Screenshot from 2023-07-26 22-54-57](https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/44624138/2f7bebda-a517-414d-b0e9-b1d52e159952)  |  ![Screenshot from 2023-07-26 18-35-05](https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/44624138/2f939c61-e41b-4f03-89f3-bca8de99b7bb)